README.md

```diff
@@ -1,16 +1,34 @@
 # gen3-client
-Gen3 client for downloading, uploading, and submitting data to Gen3 Data Commons.
+`gen3-client` is a command-line tool for downloading, uploading, and submitting data files to and from a Gen3 data commons. 
 
+Read more about what it does and how to use it in the `gen3-client` [user guide](https://gen3.org/resources/user/gen3-client/).
+
+`gen3-client` is built on Cobra, a library providing a simple interface to create powerful modern CLI interfaces similar to git & go tools. Read more about Cobra [here](https://github.com/spf13/cobra).
 
 
 ## Installation
-*TODO: update after this is made public*
+
+First, [install Go and the Go tools](https://golang.org/doc/install) if you have not already done so. [Set up your workspace and your GOPATH.](https://golang.org/doc/code.html)
+
+
+Then: 
+```
+go get -d github.com/uc-cdis/gen3-client
+go install
+```
+
+
+*TODO: Remove after GitHub repo is renamed*
+**_For now, the above actually won't work because the GitHub repo needs to be renamed. Do this instead:_**
+
 ```
 mkdir -p $GOPATH/src/github.com/uc-cdis
 cd $GOPATH/src/github.com/uc-cdis
 git clone git@github.com:uc-cdis/cdis-data-client.git
-cd cdis-data-client
+mv /cdis-data-client /gen3-client
+cd gen3-client
 go get -d ./...
 go install .
 ```
 
+If installing locally and not in a docker container, go to `gen3-cmd/root.go` and comment out the two lines that have `gitversion` and `gitcommit`. 
```

Comment:
Probably can remove this sentence. We'll fix this very soon and we don't actually want user to comment out anything.

Reply:
OK! Shall I remove the entire block above it too then?

Reply:
no we can keep the block above since renaming the repo may take some time

Reply:
👌